docs/sources/alerting/unified-alerting/alerting-rules/state-and-health.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "State and Health of alerting rules"
+description = "State and Health of alerting rules"
+keywords = ["grafana", "alerting", "guide", "state"]
++++
+
+The concepts of state and health for alerting rules help you understand, at a glance, several key status indicators about your alerts. Alert state, alerting rule state, and alerting rule health are related, but they each convey subtly different information.
+
+## Alerting rule state
+Indicates whether any of the timeseries resulting from evaluation of the alerting rule are in an alerting state. Alerting rule state only requires a single alerting instance to be in a pending or firing state for the alerting rule state to not be normal.
+- Normal: none of the timeseries returned are in an alerting state.
+- Pending: at least one of the timeseries returned are in a pending state.
+- Firing: at least one of the timeseries returned are in an alerting state.
+
+## Alert state
+Alert state is an indication of the output of the alerting evaluation engine.
+- Normal: the condition for the alerting rule has evaluated to **false** for every timeseries returned by the evaluation engine.
+- Alerting: the condition for the alerting rule has evaluated to **true** for at least one timeseries returned by the evaluation engine and the duration, if set, **has** been met or exceeded.
```

Comment:
I noticed that we have a bit of an incosistency with Prometheus here... in Prom it's `Firing`, in Grafana `Alerting`. Maybe worth to align, or mention in docs?